File: src/main/java/br/anhembi/funmodechild/model/response/ProductResponse.java
package br.anhembi.funmodechild.model.response;

import java.time.LocalDateTime;

public record ProductResponse(String id,
                              int sku,
                              String name,
                              String description,
                              double price,
                              boolean promoted,
                              CategoryResponse category,
                              LocalDateTime createdAt,
                              int quantity
) {
}

File: src/main/kotlin/br/anhembi/funmodechild/model/response/ProductResponse.kt
package br.anhembi.funmodechild.model.response

import java.time.LocalDateTime

data class ProductResponse(
    val id: String,
    val sku: Int,
    val name: String,
    val description: String,
    val price: Double,
    val promoted: Boolean,
    val category: CategoryResponse,
    val createdAt: LocalDateTime,
    val quantity: Int
)
